lib/src/main/java/com/diffplug/spotless/generic/IndentStep.java

```diff
@@ -49,32 +55,21 @@ public FormatterStep create(int numSpacesPerTab) {
 
 	/** Creates a step which will indent with the given type of whitespace, converting between tabs and spaces at the given ratio. */
 	public static FormatterStep create(Type type, int numSpacesPerTab) {
-		Objects.requireNonNull(type, "type");
```

Comment:
At first I wasn't sure about this, but then I remembered Java 14+ has [JEP 358: Helpful NullPointerExceptions](https://openjdk.org/jeps/358), and I'm not sure if punishing Java 14+ users with less helpful error messages like what we had here is worth the small drop in error message quality for users stuck on older versions.

**tl;dr**: This seems fine to me!